code/builders/builder-vite/input/iframe.html

```diff
@@ -28,6 +28,8 @@
     // doesn't fail (it will simply be disabled)
     window.module = undefined;
     window.global = window;
+    // prevent: "ReferenceError: process is not defined"
+    window.process = window.process || {};
```

Comment:
The package manager cli code shouldn't be running in the iframe, should it?  

I would not want to merge this PR with this change, but I think you're doing it just for troubleshooting.

Reply:
Actually...

I am running into this:
https://app.circleci.com/pipelines/github/storybookjs/storybook/64114/workflows/6ea5ea73-052c-4872-8329-316db3da94c3/jobs/598100

To be specific:
https://6308736456ad2046275c0ae7-hzqlbitzzd.chromatic.com

To be even more specific:
https://6308736456ad2046275c0ae7-hzqlbitzzd.chromatic.com/?path=/story/stories-renderers-svelte-args--remount-on-reset-story-args

```
process is not defined
ReferenceError: process is not defined
    at https://6308736456ad2046275c0ae7-hzqlbitzzd.chromatic.com/assets/index-cmme_mpW.js:395:7384
```

I tried bundling in the plugin, and that resulted in that error, but even after reverting that, I'm still seeing that.
I think it might just be how vite5 works?

So, I'm trying anything I can to get CI green. Been trying all day.

Reply:
And it's still broken, even with this...